benefits/core/migrations/0013_create_calitp_group.py

```diff
@@ -0,0 +1,14 @@
+from django.db import migrations
+
+
+def create_calitp_group(apps, schema_editor):
```

Comment:
Similar request as above, can we use `create_staff_group` or similar and save the Cal-ITP specifics for the value? (Same with this file's name).

Reply:
Done in https://github.com/cal-itp/benefits/pull/2220/commits/e14eae3467101738c1f466842e41a6e1fc3a1cd4